index.html

```diff
@@ -1,173 +1,272 @@
+        <div class="social-media">
+          <ul class="social-media-icon">
+            <li>
+              <a href="#"
+                ><img src="./images/Icons/github.png" alt="Github"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img src="./images/Icons/linkedin.png" alt="Linkedin"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img src="./images/Icons/Vector.svg" alt="Angellist"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img src="./images/Icons/twitter.png" alt="Twitter"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img src="./images/Icons/medium.png" alt="Medium"
+              /></a>
+            </li>
+          </ul>
+          <a class="arrow" href="#"
+            ><img src="./images/Icons/element-right.png" alt="arrow down"
+          /></a>
+        </div>
+      </section>
 
-    <section class="container">
+      <section class="container" id="portfolio">
         <div class="experience-title" id="profile">
-            <h2>MY RECENT <br> WORK</h2>
-        </div>        
-        <hr class="divider">    
-        <section class="experience-container bg1">        
-            <img src="./images/Icons/bg.jpg" alt="project image">
-            <div class="sec-wrap">
-                <h3>Multi-Post Stories</h3>
-            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+          <h2>
+            MY RECENT <br />
+            WORK
+          </h2>
+        </div>
+        <hr class="divider" />
+        <section class="experience-container bg1">
+          <img src="./images/Icons/bg.jpg" alt="project image" />
+          <div class="sec-wrap">
+            <h3>Multi-Post Stories</h3>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a standard
+              dummy text.
+            </p>
             <ul class="stack-container">
-                <li class="border-right padding-right">css</li>
-                <li class="border-right padding-right">html</li>          
-                <li class="border-right padding-right">bootstrap</li>         
-                <li class="padding-right">Ruby</li>
+              <li class="border-right padding-right">css</li>
+              <li class="border-right padding-right">html</li>
+              <li class="border-right padding-right">bootstrap</li>
+              <li class="padding-right">Ruby</li>
             </ul>
             <button type="button" class="project-button">See Project</button>
```

Comment:
Kindly work on the requested design in your buttons, hover effects. Follow the design guidelines for it on each button.